cli/UpdateTicketsCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\OtrsExtractAddress\Cli;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use WMDE\OtrsExtractAddress\DataAccess\TicketNumberReader;
+use WMDE\OtrsExtractAddress\UseCases\UpdateTickets\UpdateTicketsUseCase;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class UpdateTicketsCommand extends Command {
+
+	/**
+	 * @var UpdateTicketsUseCase
+	 */
+	private $updateTicketsUseCase;
+
+	public function __construct( UpdateTicketsUseCase $updateTicketsUseCase ) {
+		parent::__construct();
+		$this->updateTicketsUseCase = $updateTicketsUseCase;
+	}
+
+	protected function configure() {
+		$this
+			->setName( 'update-tickets' )
+			->setDescription( 'Update tickets with new owner' )
+			->setHelp( 'You must have the variables OTRS_URL, OTRS_USER and OTRS_PASSWORD set in the environment or an .env file' )
+			->addOption( 'owner', 'i', InputOption::VALUE_REQUIRED, 'OTRS Owner id', 27 )
```

Comment:
First and second `addOption()` parameters are long and short option name, `i` is a strange alias for `owner`.